changelog/std-container-dlist-linearRemoveElement.dd

```diff
@@ -0,0 +1,12 @@
+`std.container.dlist` now supports `linearRemoveElement`
+
+Now `linearRemoveElement` can be used to remove the first occurence of an element from the dlist
+
```

Comment:
`linearRemoveElement` removes the first occurence of an element fro mthe dlist

ditto slist